eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -213,170 +211,32 @@ function Upload-Blobs
         }
     } 
     else {
-        Write-Warning "Not able to do the master link replacement, since no release tag found for the release. Please manually check."
+        LogWarning "Not able to do the master link replacement, since no release tag found for the release. Please manually check."
     } 
    
-    Write-Host "Uploading $($PkgName)/$($DocVersion) to $($DocDest)..."
+    LogDebug "Uploading $($PkgName)/$($DocVersion) to $($DocDest)..."
     & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/$($DocVersion)$($SASKey)" --recursive=true --cache-control "max-age=300, must-revalidate"
 
-    Write-Host "Handling versioning files under $($DocDest)/$($PkgName)/versioning/"
+    LogDebug "Handling versioning files under $($DocDest)/$($PkgName)/versioning/"
     $versionsObj = (Update-Existing-Versions -PkgName $PkgName -PkgVersion $DocVersion -DocDest $DocDest)
 
     # we can safely assume we have AT LEAST one version here. Reason being we just completed Update-Existing-Versions
     $latestVersion = ($versionsObj.SortedVersionArray | Select-Object -First 1).RawVersion
 
     if ($UploadLatest -and ($latestVersion -eq $DocVersion))
     {
-        Write-Host "Uploading $($PkgName) to latest folder in $($DocDest)..."
+        LogDebug "Uploading $($PkgName) to latest folder in $($DocDest)..."
         & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/latest$($SASKey)" --recursive=true --cache-control "max-age=300, must-revalidate"
     }
 }
 
-if ($Language -eq "javascript")
-{
-    $PublishedDocs = Get-ChildItem "$($DocLocation)/documentation" | Where-Object -FilterScript {$_.Name.EndsWith(".zip")}
-
-    foreach ($Item in $PublishedDocs) {
-        $PkgName = "azure-$($Item.BaseName)"
-        Write-Host $PkgName
-        Expand-Archive -Force -Path "$($DocLocation)/documentation/$($Item.Name)" -DestinationPath "$($DocLocation)/documentation/$($Item.BaseName)"
-        $dirList = Get-ChildItem "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)" -Attributes Directory
-
-        if($dirList.Length -eq 1){
-            $DocVersion = $dirList[0].Name
-            Write-Host "Uploading Doc for $($PkgName) Version:- $($DocVersion)..."
-            $releaseTag = RetrieveReleaseTag "NPM" $PublicArtifactLocation
-            Upload-Blobs -DocDir "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)/$($DocVersion)" -PkgName $PkgName -DocVersion $DocVersion -ReleaseTag $releaseTag
-        }
-        else{
-            Write-Host "found more than 1 folder under the documentation for package - $($Item.Name)"
-        }
-    }
-}
 
-if ($Language -eq "dotnet")
+if ($PublishGithubIODocsFn)
```

Comment:
We blindly set the name https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/common.ps1#L33 so I'd expect that this check will always pass. What we need to do is see if the function exists.